modules/nw-ovn-kubernetes-rollback.adoc

```diff
@@ -6,17 +6,52 @@
 // This procedure applies to both a roll back and a migration
 :_mod-docs-content-type: PROCEDURE
 [id="nw-ovn-kubernetes-rollback_{context}"]
-= Migrating to the OpenShift SDN network plugin
+= Using the offline migration method to roll back to the OpenShift SDN network plugin
 
-As a cluster administrator, you can migrate to the OpenShift SDN Container Network Interface (CNI) network plugin.
-During the migration you must reboot every node in your cluster.
+As a cluster administrator, you can rollback to the OpenShift SDN Container Network Interface (CNI) network plugin by using the offline migration method. During the migration you must reboot every node in your cluster. With the offline migration method, there is some downtime, during which your cluster is unreachable. 
```

Comment:
@pliurh is the reboot manual for offline migration? When I read this, it seems like its cluster admins responsibility to reboot when i thought it was automatic for offline.

Reply:
Interesting. Our docs indicate that admins but manually reboot each node. If you "ctrl+f" "reboot", that comes up a few times: https://docs.openshift.com/container-platform/4.15/networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.html#nw-ovn-kubernetes-migration_migrate-from-openshift-sdn

Reply:
@martinkennelly According the migration steps from https://github.com/openshift/release/blob/master/ci-operator/step-registry/ovn/sdn-migration/ovn-sdn-migration-commands.sh and
https://github.com/openshift/release/blob/master/ci-operator/step-registry/ovn/sdn-migration-rollback/ovn-sdn-migration-rollback-commands.sh, cluster admins should manually rebooting the nodes during offline migration.